src/components/overlays/AlertDialog/AlertDialogZone.tsx

```diff
@@ -0,0 +1,67 @@
+import { Dialog } from './types';
+import { Portal } from '../../portal';
+import { ModalProvider } from '@react-aria/overlays';
+import { DialogContainer } from '../Dialog';
+import { AlertDialog } from './AlertDialog';
+
+export interface DialogZoneProps {
+  dialogs: Dialog[];
+}
+
+export function AlertDialogZone(props: DialogZoneProps): JSX.Element {
+  const { dialogs } = props;
+
+  return (
+    <>
+      {dialogs.map(({ props, meta }) => {
```

Comment:
Actually, it's a bad practice to use multiple modal dialogs (and AlertDialogs are always modal as far as I know). At least we can try to warn developers about the complication.